pages/docs/tutorials/create-asyncapi-document.md

```diff
@@ -0,0 +1,221 @@
+
+[MQTT](https://mqtt.org/), is a well-known protocol that is widely used in IoT applications because it was created particularly to address machine-to-machine (M2M) communication.
+
+## Create AsyncAPI document
+
+In this step, you will create an AsyncAPI document to describe the Streelights API. It will help you generate the code and the documentation later on.
+
+To create one, you can either use the [AsyncAPI Studio](https://studio.asyncapi.com) or the [AsyncAPI CLI](https://github.com/asyncapi/cli), depending on your project's needs..
+
+<Remember>
+
+You can create a new `asyncapi.yaml` document by running:
+`asyncapi new --example=tutorial.yml --no-tty`.
+
+</Remember>
+
+Go ahead to create the specification documents titled `asyncapi` with a `.yaml` extension.
+
+<CodeBlock>
+{`asyncapi: '3.0.0'
+info:
+  title: Streetlights API
+  version: '1.0.0'
+  description: |
+    The Smartylighting Streetlights API allows you
+    to remotely manage the city lights.
+  license:
+    name: Apache 2.0
+    url: 'https://www.apache.org/licenses/LICENSE-2.0'
+servers:
+  mosquitto:
+    host: test.mosquitto.org
+    protocol: mqtt
+channels:
+  lightMeasured:
+  	address: 'light/measured'
+	messages:
+		lightMeasuredMessage:
+			name: LightMeasured
+			payload:
+				type: object
+				properties:
+					id:
+						type: integer
+						minimum: 0
+						description: Id of the streetlight.
+					lumens:
+						type: integer
+						minimum: 0
+						description: Light intensity measured in lumens.
+					sentAt:
+						type: string
+						format: date-time
+						description: Date and time when the message was sent.
+operations:
+	onLightMeasured:
+		action: 'recieve'
+		summary: Inform about enviromental lighting conditions for a particular streetlight.
+		operationId: onLightMeasured
+		channel:
+			$ref: '#/channels/lightMeasured'`}
+</CodeBlock>
+
+Let's break it down into pieces:
+
+<CodeBlock>
+{`asyncapi: '3.0.0'
+info:
+  title: Streetlights API
+  version: '1.0.0'
+  description: |
+    The Smartylighting Streetlights API allows you
+    to remotely manage the city lights.
+  license:
+    name: Apache 2.0
+    url: 'https://www.apache.org/licenses/LICENSE-2.0'
+`}
+</CodeBlock>
+
+- The `asyncapi` field indicates you use the AsyncAPI version 3.0.0.
+
+- The `info` section holds information about the Streetlights API. Here, its
+title, version, description and license is defined.
+
+Moving on, let's talk about the `servers` section. In this section, you define
+the connection details of the servers that will handle the events sent by
+the streetlights to your API. Each server's connection detail is
+defined as a Server Object:
+<CodeBlock>
+{`servers:
+  	mosquitto:
+  		host: test.mosquitto.org
```

Comment:
```suggestion
  		host: test.mosquitto.org
```
Is this being changed from `mqtt://test.mosquitto.org`?
Also the following explanation under it?